tests/segarray_test.py

```diff
@@ -605,12 +605,25 @@ def test_register_attach(self):
 
         # Verify is_registered
         self.assertTrue(segarr.is_registered())
-        # Unregister one component
-        segarr.values.unregister()
-        with self.assertWarns(UserWarning):
-            segarr.is_registered()
-        # Unregister all components
         segarr.unregister()
         self.assertFalse(segarr.is_registered())
 
         self.assertEqual(len(ak.list_registry()), 0)
+
+    def test_unregister_by_name(self):
```

Comment:
Do we test all the register/attach stuff in the individual object tests? I am wondering if we should centralize this at some point to test all register/attach features in one place?

Reply:
Moved this into `registration_tests.py`